### PR TITLE
FEAT: Start LAMP Ad-Hoc Task with Github

### PR DESCRIPTION
**Pull request by rymarczy**
This change allows LAMP Ad-Hoc ECS Task to be triggered with github action.

Ad-Hoc ECS Task created by devops PR: https://github.com/mbta/devops/pull/2184

Asana Task: https://app.asana.com/0/1205827492903547/1207556073650893

### CODE REVIEW

File: .github/actions/run_task/run_task.sh [hunk 1]
```diff
@@ -11,10 +11,10 @@ set -e -u
 # - ENVIRONMENT
 
 # Get the VPC ID based on the VPC name
-if [ "$ENVIRONMENT" == "staging" ]; then
+if [ "$ENVIRONMENT" == "staging" ] || [ "$ENVIRONMENT" == "dev" ]; then
     VPC_NAME="vpc-dev-ctd-itd"
 elif [ "$ENVIRONMENT" == "prod" ]; then
-    VPC_NAME="vpc-prod-ctd-itd"A
```
Comment (arkadyan) [inline .github/actions/run_task/run_task.sh:17]: 👍

**Review by arkadyan — APPROVED** [review]
👍 

I assume the devops PR should go first before this gets merged?

**Comment by rymarczy** [comment]
> 👍
> 
> I assume the devops PR should go first before this gets merged?

Yup, I will wait to merge until the devops PR has been fully closed!

**Review by arkadyan — APPROVED** [review]
👍 Makes sense

**Pull request merged**